core/blockchain.go

```diff
@@ -174,6 +175,7 @@ var DefaultCacheConfig = &CacheConfig{
 	CommitInterval:        4096,
 	AcceptorQueueLimit:    64, // Provides 2 minutes of buffer (2s block target) for a commit delay
 	SnapshotLimit:         256,
+	AcceptedCacheSize:     32,
```

Comment:
We may want to increase this default substantially. It isn't a lot of data AFAICT.